### ISSUE TITLE
The 5e Default Invisible status effect doesn't work With CLT Enabled

### ISSUE THREAD

**Issue by GameCrazed** [mclemente/fvtt-condition-lab-triggler#38]
**Describe the bug**
With CLT enabled with the config in the attached json, checking the Invisible field on a token or character sheet does nothing. Sometimes it will apply the 'invisible' visual effects to the token but won't actually make them invisible to other players 

**To Reproduce**
1) Import the included config
2) select token.
3) choose the "Invisible" status effect.
4) See nothing happens / the token's visuals change but they aren't actually made invisible to other players.

**Expected behavior**
I'd expect the token to be made invisible to other players unless they had some kind of sense configured to see them (ie tremoursense, true sight, etc).

**Screenshots**
[cub-dnd5e-conditions.json](https://github.com/mclemente/fvtt-condition-lab-triggler/files/14577427/cub-dnd5e-conditions.json)

Not sure if the config json includes this:
![image](https://github.com/mclemente/fvtt-condition-lab-triggler/assets/7393464/56fd0b10-55e6-4aa8-afc9-37348d6bef2c)

**Environment Info (please complete the following information):**
 - Foundry Version: [11.315]
 - Game System / Version: dnd5e: [3.0.4]
 - CUB Version [1.5.4]
 - Browser [Chrome and Firefox] (tested in both)
 - OS [Forge] (forge hosted but also tested locally on a Windows 10 machine)

**Additional context**
N/A

**Comment by strongpauly** [comment]
I wasn't able to reproduce this with the latest (1.5.5) CLT.  Are you sure the selected token (that was supposed to not see the "Invisible" token) had vision enabled?

**Comment by GameCrazed** [comment]
So this is an interesting (read: annoying) one. For clarity I'm having this issue on my main game hosted on the Forge. That being said, if I create a brand new world on the Forge, enable ONLY this module, create a PC/NPC, throw them on a map, and hit the invisible AE, they get their blueish invisible overlay as expected.

However, if I go to my main game where I actually _need_ it, disable every mod EXCEPT this (and the Forge module but that's mandatory for all Forge-hosted sessions so shouldn't be affecting anything, it was present in both tests). WHen I tell a toke to go invisible it applies the 'invisible' icon on the token & sheet, but doesn't apply the blue overlay or any of the invisible token behavious (see image below).

At this point I'm not sure why it's happening. I've tried uninstalling the module entirely and re-downloading & configuring but that doesn't seem to do anything. I've also tried cloning my game world to see if the issue persists (it does). If you've any ideas on what I could try here I'm happy to debug things, as you'll see in the screenshot, nothing's getting reported to the console at present to indicate any kind of issue.

At point of image, the Invisible option has just been selected for the Hunter token
![image](https://github.com/mclemente/fvtt-condition-lab-triggler/assets/7393464/1fbdc478-55f4-484b-9164-fe09e5a63ce4)

I also neglected to drop my config for Triggler in my initial post, apologies: 
[cub-triggers.json](https://github.com/mclemente/fvtt-condition-lab-triggler/files/14720748/cub-morterrus-triggers.json)

**Comment by GameCrazed** [comment]
Good thinking, I'll try that fix now. Here is my CONFIG.specialStatusEffects before I make the attempt:

`{
    "DEFEATED": "dead",
    "INVISIBLE": "condition-lab-triggler.invisible",
    "BLIND": "blind"
}`

Comparing that to my localhost test which _does_ work it looks like INVISIBLE should be "invisible"...

And trying your fix, well, **fixed it!** 

My best guess would be that prior to D&D5e v3, I had every status manually defined in Condition Lab and overiding the defaults, including the Invisible condition. Then when v3 hit I found there was a bunch of errors happening so I scaled it back to not override the values and knocked out most of the conditions until it was what you see in the config files. My guess would be that during that process, something got screwed up with the configs.

Either way, thank you so much for the fix suggestion there, this thing's been driving me nuts for weeks!
